benchmarks/benchmarks.c

```diff
@@ -55,27 +55,52 @@ void benchmarkFillup (void)
 	}
 }
 
+
 /**
  * Arbitrary Key Set Generator
  */
 
-static size_t shapefDefault (size_t initSize ELEKTRA_UNUSED, size_t size, size_t level, int32_t * seed);
```

Comment:
@KurtMi: 👋 Seems like you never used the argument `size` inside this function:

```
libelektra/benchmarks/benchmarks.c:570:73: error: unused parameter 'size' [-Werror,-Wunused-parameter]
static void shapefDefault (const size_t initSize ELEKTRA_UNUSED, size_t size, size_t level, int32_t * seed ELEKTRA_UNUSED,
                                                                        ^
1 error generated.
```

. Currently three of the Travis build jobs [fail](https://travis-ci.org/ElektraInitiative/libelektra/jobs/278224595#L815) because of this (minor) issue.